extensions/libraries/redcore/media/redcore/translations/com_banners/banners.xml

```diff
@@ -14,7 +14,7 @@
             <field type="state" name="state" translate="1" alwaysFallback="true">State</field>
         </table>
         <component>
-            <form>com_banners#banners#id#task#!edit</form>
+            <form>com_banners#banner#id#task#!edit</form>
```

Comment:
didnt we agree we will change this pattern?
instead of  <component>
           <form>com_contact#something#cid#!edit</form>
       </component>
it should be:
<component>
<com_banners admin="true/false"><banner><edit><id></id></edit></banner></com_banners>
</component>
Generic would be
<component>
<component_name admin="true/false"><view_name><layout_name><id_name></id_name></layout_name></view_name></component_name>
</component>